### PR TITLE
Single GCP container

### PR DESCRIPTION
**Pull request by MitchellSlavik**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+FROM node:10
```
Comment (ColinKrist) [inline Dockerfile]: reason for node 10?

File: zoo_frontend/src/api/Animals.js [hunk 2]
```diff
@@ -1,6 +1,10 @@
 import axios from 'axios';
 
-const API_BASE_URL = process.env.BACKEND_URL;
+let API_BASE_URL = process.env.BACKEND_URL;
+
+if (typeof window === 'undefined') {
+  API_BASE_URL = `http://localhost:${process.env.PORT}`;
+}
```
Comment (ColinKrist) [inline zoo_frontend/src/api/Animals.js]: would it be better to stick this into a /util file and import this variable?

File: Dockerfile (hunk 1 above)
Comment (MitchellSlavik) [reply Dockerfile]: ¯\_(ツ)_/¯

File: zoo_frontend/src/api/Animals.js (hunk 2 above)
Comment (MitchellSlavik) [reply zoo_frontend/src/api/Animals.js]: probably

**Pull request merged**